deploy.json

```diff
@@ -241,10 +241,7 @@
       "dst": "[[stasis]]/integrations/",
       "match": "^.*\\.(py)$",
       "recursive": true
-    },
-
-    "// run unit and coverage tests",
-    {"type": "py3test"}
```

Comment:
removing this will prevent `github-deploy-repo` from running local tests during its deployment process